eclcli/vnf/v1/virtual_network_appliance.py

```diff
@@ -271,6 +279,36 @@ def take_action(self, parsed_args):
 
                 interface_object.update(tmp)
 
+        initial_config_object = None
+        if parsed_args.initial_config:
+            config_dict = {"format": None, "data": None}
+            for item in parsed_args.initial_config.split(","):
+                parts = item.split('=')
+                if len(parts) == 2:
+                    config_dict[parts[0].strip()] = parts[1].strip()
+
+            config_format = config_dict.get('format')
+            config_path = config_dict.get('data')
+            if not config_format or not config_path:
+                msg = _("You must specify both initial_config.format and initial_config.data (e.g., --initial-config format=set,data=/path/to/config)")
+                raise exceptions.CommandError(msg)
+            if config_format not in ['set', 'text']:
+                msg = _("initial_config.format must be 'set' or 'text'")
+                raise exceptions.CommandError(msg)
+
+            config_data = None
+            try:
+                with open(config_path, 'rb') as f:
+                    config_data = base64.b64encode(f.read()).decode()
+            except IOError as e:
```

Comment:
【MUST】
例外処理が不十分かつ不正確と思われます。

IOError 例外は必ずしもファイルが存在しないことを意味しません。
たとえば実行権限不足でファイルが開けない場合なども含まれます。

Ref: https://peps.python.org/pep-3151/#appendix-a-survey-of-common-errnos

また、 IOError は Python 3.3 で OSError に統合されました。

Ref: https://docs.python.org/ja/3/library/exceptions.html

ECLCLI は python 3.7-3.9 依存のようなので、 OSError で受け止めるのがよいと思われます。

Ref: https://sdpf.ntt.com/services/docs/eclc/tutorials/rsts/installation.html#:~:text=Python%203.7%20%E4%BB%A5%E9%99%8D%E3%81%8B%E3%81%A4%203.9%20%E4%BB%A5%E4%B8%8B%E3%82%92%E3%82%A4%E3%83%B3%E3%82%B9%E3%83%88%E3%83%BC%E3%83%AB%E3%81%97%E3%81%A6%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84%E3%80%82


以下のような修正でどうでしょう。

```suggestion
            except OSError as e:
                msg = "Failed to open/read config file %s: %s"
                raise exceptions.CommandError(msg % (config_path, e))
```

Reply:
参考元となった過去mLBのcommitと合わせて修正済み